src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir.ttc);
+}
+
+.searchForm {
+  width: 100%;
+  position: relative;
+  color: rgb(61, 78, 97);
+  font-weight: 900;
+  font-family: Avenir, Arial, sans-serif;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchForm::placeholder {
+  color: rgb(61, 78, 97);
+  font-weight: 400;
+}
+
+.searchForm:focus {
+  border: 1px solid rgb(225, 231, 237);
+  border-radius: 3px;
+  background: linear-gradient(180deg,
+  rgb(255, 255, 255) 0%,
+  rgb(246, 246, 247) 100%);
+}
+
+.searchForm:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+html {
+  font-weight: 300;
+}
+
+.big {
+  margin-top: 12px;
+  font-size: 16px;
+  line-height: 23px;
+  padding: 22.5px 0;
+  text-indent: 61px;
+  margin-bottom: 20px;
+  box-shadow: none;
+}
+
+.small {
+  padding: 11px 0;
+  text-indent: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.image {
+  position: relative;
+  z-index: 1;
+}
+
+.image1 {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: -107px;
+}
+
+.image2 {
+  left: -10px;
+  top: -35px;
+  height: 11px;
+  width: 11px;
+}
```

Comment:
next time do according to the checklist point 2
https://mateacademy.slack.com/archives/C02M6400QF6/p1637221010303800